test/__snapshots__/katex-spec.js.snap

```diff
@@ -88,7 +88,7 @@ exports[`An implicit group parser within optional groups should work with \\colo
 ]
 `;
 
-exports[`An implicit group parser within optional groups should work with sizing commands: \\sqrt[\\small 3]{x} 1`] = `
+exports[`An implicit group parser within optional groups should work with old font functions: \\sqrt[\\tt 3]{x} 1`] = `
```

Comment:
It looks like the order of these changed, but the parse trees look the same.

Reply:
I fixed a typo, assuming that's why?

Reply:
Sorry, I didn't mean to suggest that anything needed changing here.